shared/uavobjectdefinition/inssettings.xml

```diff
@@ -18,6 +18,6 @@
 		<access gcs="readwrite" flight="readwrite"/>
 		<telemetrygcs acked="true" updatemode="onchange" period="0"/>
 		<telemetryflight acked="true" updatemode="onchange" period="0"/>
-		<logging updatemode="manual" period="0"/>
+		<logging updatemode="manual" period="1"/>

```

Comment:
Yes, I agree, it's not very elegant. An option could be to use "0" for log always, and "-1" for no logging.